patterns/multisig/MultisigRune.ts

```diff
@@ -155,3 +157,17 @@ export class MultisigRune<out C extends Chain, out U> extends PatternRune<Multis
 export class NoProposalError extends Error {
   override readonly name = "NoProposalError"
 }
+
+function compare(a: Uint8Array, b: Uint8Array): number {
```

Comment:
Pretty sure the same thing that exports `equals` has a `compare`

Reply:
this? https://deno.land/std@0.185.0/bytes/mod.ts

Reply:
Doesn't seem such a fn exists for `Uint8Array`s. Could we move this fn into a new util file? What shall we name it? `compareBytes`, `compareBuf`, ...

Reply:
Can just do `bytes` and throw all the missing byte related functions in there as they come up

Reply:
Good thinking. Would we want to `export * as bytes from "./bytes.ts"` in `util/mod.ts`?

Reply:
Yeah i think that would be good, since `compare` is so generic its good to namespace it.